cmake/TorchConfig.cmake.in

```diff
@@ -69,6 +69,11 @@ if(NOT @BUILD_SHARED_LIBS@)
     list(APPEND TORCH_LIBRARIES ${XNNPACK_LIBRARY})
   endif()
 
+  if(NOT @USE_INTERNAL_PTHREADPOOL_IMPL@)
+    find_library(PTHREADPOOL_LIBRARY pthreadpool PATHS "${TORCH_INSTALL_PREFIX}/lib")
+    list(APPEND TORCH_LIBRARIES ${PTHREADPOOL_LIBRARY})
+  endif()
```

Comment:
This is for installations of PyTorch.  We may or may not end up having a dependency on the third party pthreadpool implementation.  In case we do, we need to add it to the list of libraries a client of PyTorch should link with.